contracts/SpokePool.sol

```diff
@@ -1061,8 +1187,23 @@ abstract contract SpokePool is
         );
     }
 
-    // @dev We pack the function params into USSSlowFill to avoid stack-to-deep error that occurs
-    // when a function is called with more than 13 params.
+    /**
+     * @notice Executes a slow relay leaf stored as part of a root bundle relayed by the HubPool.
+     * @dev Executing a slow fill leaf is equivalent to filling the relayData so this function cannot be used to
+     * double fill a recipient. The relayData that is filled is included in the slowFillLeaf and is hashed
+     * like any other fill sent through fillUSSRelay().
+     * @dev There is no relayer credited with filling this relay since funds are sent directly out of this contract.
+     * @param slowFillLeaf Contains all data necessary to uniquely identify a relay for this chain. This struct is
+     * hashed and included in a merkle root that is relayed to all spoke pools.
+     * - relayData: struct containing all the data needed to identify the original deposit to be slow filled.
+     * - chainId: chain identifier where slow fill leaf should be executed. If this doesn't match this chain's
+     * chainId, then this function will revert.
+     * - updatedOutputAmount: Amount to be sent to recipient out of this contract's balance. Can be set differently
+     * from relayData.outputAmount to charge a different fee because this deposit was "slow" filled. Usually,
+     * this will be set higher to reimburse the recipient for waiting for the slow fill.
```

Comment:
So the updatedOutputAmount is basically selected by the proposer and verified via the standard proposal lifecycle? We'll need to track this for an UMIP update.